technical_onboarding/github_flow.md

```diff
@@ -10,7 +10,8 @@ In short words, we are forking repository from Code for Poznan organisation to o
 
 1. Fork repository from Code for Poznań
 2. Clone your (forked) repository to local computer
-3. Add upstream remote by executing command `git remote add upstream https://github.com/CodeForPoznan/project_name.git`
+3. Go to (cd command) directory of cloned repository in a terminal
```

Comment:
use \` around `cd` to visually diff command from a sentence.

You can even write there `cd project_name` as we use `project_name` as a placeholer for a project name :)